tests/console/rsync.pm

```diff
@@ -39,7 +40,15 @@ sub run {
     my $md5_initial_sh = script_output('md5sum /tmp/rsync_test_folder_a/rsync_test_sh.sh');
     my $md5_initial_tar = script_output('md5sum /tmp/rsync_test_folder_a/rsync_test_tar.tar');
 
-    enter_cmd("rsync -avzr /tmp/rsync_test_folder_a/ root\@localhost:/tmp/rsync_test_folder_b; echo \$\? > /tmp/rsync_return_code.txt");
+    # we have an issue with 'authenticate the localhost' on svirt backend
+    if (is_svirt_except_s390x) {
+        my $filename = "/root/.ssh/known_hosts";
+        assert_script_run("touch $filename") unless (-f $filename);
```

Comment:
@Zaoliang,
I think we can remove this line :) we don't need to create it

Reply:
This is not yet resolved.